src/passes/PostEmscripten.cpp

```diff
@@ -19,6 +19,7 @@
 // emscripten output.
 //
 
+#include "wasm-emscripten.h"
```

Comment:
seems odd some includes use `"` while others use `<`. We don't have a good rule for this, but it feels weird in a single file...

Also the `<wasm*` ones are sorted near the bottom, so this should go there I think.